server/pbench/bin/pbench-inotify

```diff
@@ -176,17 +176,22 @@ class EventHandler(pyinotify.ProcessEvent):
     """
 
     def process_IN_CLOSE_WRITE(self, event):
-        filename = event.pathname.split("/")[-1]
-        if DEBUG:
-            print("\nTRIGERRED BY: %s" % filename)
-        script_to_run = associations[filename].script
-        args = associations[filename].args
-        list = open(associations[filename].list_file, 'r')
-        fcntl.flock(list, fcntl.LOCK_EX)
-        new_links = list.readlines()
-        fcntl.flock(list, fcntl.LOCK_UN)
-        list.close()
-        associations[filename].compare_and_dispatch(new_links)
+        if event.path == "/pbench/archive/fs-version-001":
```

Comment:
No hardwired  paths please (and 001 is wrong). If we need the complete path, we should get it from the config file. If not we should match on as little as possible (perhaps "fse-version-002", or even "002"). Even that should come from the config file. Do we need to match on any version *greater or equal* to 002?